src/arch/helperadvsimd.h

```diff
@@ -28,6 +28,16 @@
 
 #define ISANAME "AArch64 AdvSIMD"
 
+#ifdef SLEEF_SINGLE_MINMAXNUM_AVAILABLE
+#undef SLEEF_SINGLE_MINMAXNUM_AVAILABLE
```

Comment:
This `#ifdef`-guarded `#undef` is not what we want here. If for any reason the  `SLEEF_SINGLE_MINMAXNUM_AVAILABLE` macro is already defined when the C preprocessor reaches this point, we _do want_ to have an error here as it means that there is something wrong in the code, as the `SLEEF_SINGLE_MINMAXNUM_AVAILABLE` macro should be defined only once in all the compilation.
The preprocessor fires a warning in case of duplicate macro, but I think that an `#error` preprocessor invocation would be more appropriate.